### PR TITLE
Update post clone script

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] ~Has `CHANGELOG.md` been updated?~
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests if this modifies the Windows code?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Related PRs or issues

[AB#84802](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/84802)

### CODE REVIEW

File: appcenter-post-clone.ps1 [hunk 1]
```diff
@@ -4,4 +4,27 @@
...
+"      <add key=""ClearTextPassword"" value=""NUGET_PASSWORD"" />`n" +
+"    </VSTSAC>`n" +
+"  </packageSourceCredentials>`n" +
+"</configuration>`n"
+
+if (Test-Path $nugetFileName) {
+  Remove-Item $nugetFileName
+}
+New-Item -Path . -Name $nugetFileName -Value $contentValue
 (Get-Content NuGet.config).replace('NUGET_PASSWORD', $env:NUGET_PASSWORD) | Set-Content NuGet.config
```
Comment (MatkovIvan) [inline appcenter-post-clone.ps1:30]: Let's replace it during generating

File: appcenter-post-clone.ps1 [hunk 2]
```diff
@@ -4,4 +4,26 @@
...
+"      <add key=""Username"" value=""mobilecenter"" />`n" +
+"      <add key=""ClearTextPassword"" value=""$env:NUGET_PASSWORD"" />`n" +
+"    </VSTSAC>`n" +
+"  </packageSourceCredentials>`n" +
+"</configuration>`n"
+
+if (Test-Path $nugetFileName) {
+  Remove-Item $nugetFileName
+}
+New-Item -Path . -Name $nugetFileName -Value $contentValue
```
Comment (MatkovIvan) [inline appcenter-post-clone.ps1:29]: BTW can we just use [`nuget add source`](https://docs.microsoft.com/ru-ru/dotnet/core/tools/dotnet-nuget-add-source) command?

File: appcenter-post-clone.ps1 [hunk 3]
```diff
@@ -4,4 +4,26 @@
 # Licensed under the MIT License.
 
 echo "Executing post clone script in $pwd"
-(Get-Content NuGet.config).replace('NUGET_PASSWORD', $env:NUGET_PASSWORD) | Set-Content NuGet.config
+
+$nugetFileName = "NuGet.config"
+$contentValue = "<?xml version=""1.0"" encoding=""utf-8""?>`n" +
```
Comment (guperrot) [inline appcenter-post-clone.ps1:9]: Could probably be simplified using cat as shown in https://stackoverflow.com/a/23930212/598520

Comment (MatkovIvan) [reply appcenter-post-clone.ps1:9]: It's PowerShell, not bash, so syntax will be different

File: .gitignore [hunk 4]
```diff
@@ -278,3 +278,4 @@ SDK/**/*.nuspec
 SDK/**/*.snk
 Tests/**/*.snk
 *.snk
+NuGet.config
```
Comment (MatkovIvan) [inline .gitignore:281]: It doesn't look required anymore

Comment (AnastasiaKubova) [reply .gitignore:281]: We removed this file, and to avoid random addition this file in the future it should be here.

**Review by MatkovIvan — APPROVED** [review]

**Review by guperrot — APPROVED** [review]

**Pull request merged**